frontend/src/containers/PublicGroup.js

```diff
@@ -52,155 +48,72 @@ export class PublicGroup extends Component {
     };
   }
 
-  donationSection() {
+  _donationFlow() {
     const {
-      group,
       isAuthenticated,
-      donationForm,
       showPaypalThankYou
     } = this.props;
 
-    if (this.state.showThankYouMessage || (isAuthenticated && this.state.showUserForm) || showPaypalThankYou) { // we don't handle userform from logged in users) {
-      return <PublicGroupThanks />;
+    if (this.state.showThankYouMessage || (isAuthenticated && this.state.showUserForm) || showPaypalThankYou) {
+      return (
+        <div className='PublicGroupDonationFlowWrapper px2 py3 border-box fixed top-0 left-0 right-0 bottom-0 bg-white'>
+          <PublicGroupThanksV2
+            message='You are now in our backers wall!'
+            closeDonationModal={::this._closeDonationFlow} />
+        </div>
+      );
     } else if (this.state.showUserForm) {
-      return <PublicGroupSignup {...this.props} save={saveNewUser.bind(this)} />
-    } else {
-      return <Tiers tiers={group.tiers} {...this.props} form={donationForm} onToken={donateToGroup.bind(this)} />
-    }
-  }
-
-  expenseList() {
-    const {
-      expenses,
-      users,
-      group
-    } = this.props;
-
-    const emptyState = (
-      <div className='PublicGroup-emptyState'>
-        <div className='PublicGroup-expenseIcon'>
-          <Icon type='expense' />
+      return (
+        <div className='PublicGroupDonationFlowWrapper px2 py3 border-box fixed top-0 left-0 right-0 bottom-0 bg-white'>
+          <PublicGroupSignupV2 {...this.props} save={saveNewUser.bind(this)} />
```

Comment:
Personally, I would like to use the bind-operator when we don't need to pass any argument, just because is shorter, no other reason really. What do you think? should I replace the `::`s with `bind` for consistency?